src/main/java/net/imglib2/algorithm/metrics/imagequality/MSE.java

```diff
@@ -0,0 +1,55 @@
+package net.imglib2.algorithm.metrics.imagequality;
```

Comment:
Can you add license headers by running `mvn license:update-file-header license:update-project-license`?